approval_frame/settings.py

```diff
@@ -9,7 +9,10 @@
 SECRET_KEY = 'jndet@)ed9ge4+!k605hc+$0tjb8u5oe)pdjab)-b3lsot_xnf'
```

Comment:
Since we are generating a secret key from scratch in case it does not exist, should this line be removed from the settings altogether ?